assets/src/blocks/Gallery/GalleryCarousel.js

```diff
@@ -125,9 +125,8 @@ export const GalleryCarousel = ({ images, onImageClick, isEditing }) => {
               style={{ objectPosition: image.focus_image }}
               alt={image.alt_text}
               title={image.alt_text}
-              onClick={() => {
-                onImageClick(index);
-              }}
+              data-index={index}
```

Comment:
Doing [this](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/932/files#diff-e26889ca28b19eea2169eb16c79aee621cffa40638ad1658c76bbe66ced7c9cfL128) it will pass a `data-` value and retrieve it by doing [this](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/932/files#diff-9e3803481782cf4a26388c314322e5f985e88403feb7623a5cf459195d8d1c36R10).